content/posts/unpublished/liquibase-as-an-init-container/post.md

```diff
@@ -0,0 +1,154 @@
+# Liquibase as an Init Container
+
+Exploring Liquibase's integration as an init container, emphasizing database change locking and initialization container usage for streamlined schema management in containerized environments.
+
+![](assets/thumbnail.png)
+
+Authors: Furkan Aksin
+Date: unpublished
+Category: backend
+
+tags: liquibase, backend, init container, container, kubernetes, docker, auxiliary, config, database, change
+
+---
+
+## **Understanding Init Containers**
+
+Init containers in Kubernetes are specialized containers that run before the main application containers start. They handle initialization tasks, such as setting up configurations or fetching necessary data. This ensures that the main application containers only start once all dependencies are met.
+
+---
+
+## **Integrating Liquibase**
+
+Liquibase, a powerful database schema management tool, can be seamlessly integrated into Kubernetes environments as an init container. By leveraging Liquibase's capabilities, teams can automate database schema changes and ensure consistency across deployments.
```

Comment:
Just like you did the introduction of init container, have section on liquibase introduction as well with official documentation link